SpellChecker.pro

```diff
@@ -1,4 +1,7 @@
 QT      += core gui widgets
+greaterThan(QT_MAJOR_VERSION, 5) {
+  QT    += core5compat
```

Comment:
Which used API is not supported in Qt6? i.e. if you remove this section what error do you get?

Reply:
QTextCodec is not existing anymore in Qt6. Replacement should be QStringConverter ([see Qt blog post](https://www.qt.io/blog/porting-from-qt-5-to-qt-6-using-qt5compat-library)), but for simple compatibility for Qt 5 **and** 6 with the same source code, the most easiest solution is to add "core5compat".
If I find some time, I'll check if there is a "pure" Qt 6 solution with above mentioned new class as well, but I think for the meantime using core5compat is fine.

Reply:
> If I find some time, I'll check if there is a "pure" Qt 6 solution with above mentioned new class as well, but I think for the meantime using core5compat is fine.
Leave as it is now, I was just curious. Thanks for the contribution!